packages/main/src/themes/Token.css

```diff
@@ -19,14 +19,14 @@
 :host(:not([readonly]):hover) {
 	background: var(--sapButton_Hover_Background);
 	border-color: var(--_ui5_token_hover_border_color);
-	box-shadow: var(--sapContent_Interaction_Shadow);
+	box-shadow: var(--_ui5_token_box_shadow);
```

Comment:
Do we need that box-shadow? I do not see in the specification of any theme that we need it.

Reply:
none of the specs contain it, therefore I am moving it.